### DISCUSSION TITLE
param Chen

### DISCUSSION THREAD

**Discussion by zeecode07** [pybamm-team/PyBaMM#3212] [Q&A]
i have 2 question:
1. is it possible to use "SEI":"solvent-diffusion limited" in parameter_values Chen2020? 
2. is it possible to use "SEI":"ec reaction limited" in parameter_values OKane2022?
 @tinosulzer @brosaplanella

**Comment by valentinsulzer** [comment]
Yes but you might need to update the parameter sets with parameters specifically for those SEI models (e.g. using the ones given in Marquis2019). These are only representative, not fitted for that particular cell

**Reply by zeecode07** [reply]
what values to update "SEI":"ec reaction limited" in parameter_values OKane2022?

**Reply by brosaplanella** [reply]
You can call `model.print_parameter_info()` to know which parameters does your model need.
